sdk/identity/identity/review/identity.api.md

```diff
@@ -56,7 +56,7 @@ export class ChainedTokenCredential implements TokenCredential {
 
 // @public
 export class ClientCertificateCredential implements TokenCredential {
-    constructor(tenantId: string, clientId: string, certificatePath: string, options?: TokenCredentialOptions);
+    constructor(tenantId: string, clientId: string, certificatePath: string, options?: TokenCredentialOptions, sendX5c?: boolean);
```

Comment:
The pattern we use for options in Identity credential classes is to create a new interface that extends `TokenCredentialOptions` and adds the additional options, like this one:

https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/src/credentials/interactiveBrowserCredentialOptions.ts#L19

You'd then change the constructor of this interface to use that options type for the `options` parameter.  I'd recommend naming the option `includeX5c` unless @schaabs has a better name.